docs/types.rst

```diff
@@ -47,58 +47,34 @@ represent TEXT columns.  This was chosen for maximum forward compatibility with
 Python 3, and to make it easier to write code that will work identically in
 both languages.  This decision has many important ramifications.
 
-#.  In Python 2, ``'foo'`` is the wrong type for binding a ``cstring`` column.
-    You instead need to use a Unicode literal like ``u'foo'``.
-    Alternately, you can make all string literals in your module be Unicode
-    literals by default by doing::
+#.  If you have a variable of type `bytes` and you want to pass it to the
+    database as a TEXT value, you need to convert it to a `str` using
+    `~bytes.decode`.
 
-        from __future__ import unicode_literals
-
-    .. note::
-        It's not safe to blindly do this in existing code as it changes that
-        code's behavior, and might cause problems.
-
-#.  If you have a variable  of type `six.binary_type` (byte string) and you
-    want to pass it to the database as a TEXT value, you need to convert it to
-    a `six.text_type` (Unicode) string using `~bytes.decode`.  In Python 2,
-    unless ``unicode_literals`` was imported from `__future__`, you will need to
-    do this with every string variable that you want to use as a cstring
-    column.  For example::
-
-        value = 'foo'
-        params = {'col': value.decode('utf-8')}
-        # dbapi2
-        cursor.execute("select * from tbl where col=%(col)s", params)
-        # cdb2
-        handle.execute("select * from tbl where col=@col", params)
-
-#.  TEXT columns are returned to you as `six.text_type` (Unicode) strings.
-    If you need to pass them to a library that expects `six.binary_type` (byte)
-    strings, you have to use `~str.encode`, like this::
+#.  TEXT columns are returned to you as `str` strings. If you need to pass them
+    to a library that expects `bytes` strings, you have to use `~str.encode`,
+    like this::
```

Comment:
Let's just drop this bullet (and the example that goes with it). These days libraries that expect strings to be passed to them as `bytes` are practically nonexistent in the Python 3 world, so this example probably makes things less clear for readers, not more clear.